src/server.c

```diff
@@ -338,8 +339,9 @@ static int curl_get_file(const char *url, const char *prefix, time_t timestamp)
 
                 free(command); /* reuse */
                 if (asprintf(&command,
-                             "tar -C /var/cache/debuginfo/%s --no-same-owner "
+                             "tar -C %s/%s --no-same-owner "
                              "--no-same-permissions -xf %s",
+                             CACHE_DIR,
```

Comment:
These changes looks fine, but the hardcoded paths in `main()` also need to be updated (line 477).

Reply:
sorry I didn't see them, changing